src/parser/jobs/vpr/modules/SerpentsIre.tsx

```diff
@@ -0,0 +1,56 @@
+import {t, Trans} from '@lingui/macro'
+import {DataLink} from 'components/ui/DbLink'
+import {EvaluatedAction, ExpectedActionsEvaluator, TimedWindow} from 'parser/core/modules/ActionWindow'
+import {HistoryEntry} from 'parser/core/modules/ActionWindow/History'
+import {SEVERITY} from 'parser/core/modules/Suggestions'
+import React from 'react'
+import DISPLAY_ORDER from './DISPLAY_ORDER'
+
+const SEVERITY_TIERS = {
+	1: SEVERITY.MEDIUM,
+	2: SEVERITY.MAJOR,
+}
+
+const OPENER_BUFFER = 30000 //After 30 seconds, you have reached the point that you could do 2 reawakens if you burst right there.
+
+export class SerpentsIre extends TimedWindow { // AKA Peusdo 2 Minute Window
+	static override handle = 'serpentsIre'
+	static override title = t('vpr.serpents_ire.title')`"2 Minute Windows"`
```

Comment:
I get why you had this title in sarcasm quotes, but I don't know that it'll translate well to other languages. I feel like just calling it Serpent's Ire would be enough.

Reply:
fair enough, I think with a prepend message like you suggested in another comment it'll explain itself well enough if we just call it Serpent's Ire